user_input.py

```diff
@@ -6,9 +6,17 @@
 
 
 def handle_key_input(user_input):
-    """Use tdl's user input features to react to keyborad input.
+    """
+    Use tdl's user input features to react to keyborad input.
+
+    Args:
+        user_input (tdl.Event): An Event object from the tdl library that
```

Comment:
The way it's coded now, since it's for handling key input, you can actually go further and say that `user_input` is a `tdl.event.KeyDown` object, which is a subclass of `tdl.event.KeyEvent`, which is a subclass of `tdl.event.Event`. It actually checks for this in *main*, but will change when I refactor it into an action manager class.